### ISSUE TITLE
No alt text for refresh button in smart fitler bar 

### ISSUE THREAD

**Issue by truezn** [SAP/fundamental-ngx#10182]
#### Is this a bug, enhancement, or feature request?
Bug in accessibility 
#### Briefly describe your proposal.
During our accessibility test, we have found that there no alternative text for refresh button in smart filter bar 
![image](https://github.com/SAP/fundamental-ngx/assets/10548281/cb797e5f-8bb2-4af0-892e-f8182cd6ebd7)
#### Which versions of Angular and Fundamental Library for Angular are affected? (If this is a feature request, use current version.)
Angular: 14
Fundamental: v0.36.0
#### If this is a bug, please provide steps for reproducing it.
It is accessibility bug.
#### Please provide relevant source code if applicable.
![image](https://github.com/SAP/fundamental-ngx/assets/10548281/10d541ff-1278-43da-81d9-64f7fe88c26e)
#### Is there anything else we should know?
No anything else
